sound/soc/sof/topology.c

```diff
@@ -3005,6 +3048,11 @@ static void sof_complete(struct snd_soc_component *scomp)
 			break;
 		}
 	}
+	/*
+	 * cache initial values of SOF kcontrols by reading DSP value over
+	 * IPC. It may be overwritten by alsa-mixer after booting up
+	 */
+	snd_sof_cache_kcontrol_val(sdev);
```

Comment:
question: do we deal with this cache at the first boot or every time we resume (runtime_pm or not)?

Reply:
@plbossart I think it is enough if we do this at first boot. When we resume, we actually do the opposite where we send the cached values to the DSP